WORKSPACE

```diff
@@ -38,6 +38,12 @@ load("//builder/java:deps.bzl", java_deps = "deps")
 java_deps()
 load("//library/maven:rules.bzl", "maven")
 
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
+rules_jvm_external_deps()
+
+load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
+rules_jvm_external_setup()
+
```

Comment:
Is this necessary in dependencies? Why didn't we have that here before?

Reply:
Sorry, this comment should have been part of the self-review.
It's the instructions here: https://github.com/bazelbuild/rules_jvm_external/releases/tag/5.3
And also synced up with what @flyingsilverfin is doing in his PR : https://github.com/vaticle/dependencies/pull/454

Reply:
^ Thank you, you're right. We don't need it in dependencies.
I've removed it now.